Tools/Scripts/libraries/webkitcorepy/webkitcorepy/environment.py

```diff
@@ -55,6 +57,17 @@ def get(self, key):
             return os.environ[key]
         return self._mapping.get(key)
 
+    def secure(self, *paths):
```

Comment:
Can you add a docstring here the clarify what this function is doing?

Reply:
`secure` doesn't seem to convey the message that the credential file will be deleted or even the opposite info. What it does is remove the credential files from disk and only keep them in memory. However, I cannot think of a concise term for it. Maybe we can keep it `secure` for now.

Also, what this function does is not quite clear to me. It removes all credentials under current environment + the files specified by the `paths`. Maybe renaming `paths` to `extra_paths`?

Reply:
Just a short note, this function doesn't delete credentials that are in memory, it deletes credentials that are _not_ in memory. This is because services often have different processes that expect credentials to be on disk. However, we don't want to keep all credentials around when we start up a web server, particularly those that aren't actually in use but are included in whatever Kubernetes image we've mounted.